pkgs/by-name/ya/yazi-unwrapped/package.nix

```
@@ -59,6 +60,7 @@ rustPlatform.buildRustPackage rec {
   '';
 
   passthru.updateScript.command = [ ./update.sh ];
```

I think you can replace this with `nix-update-script` and remove the `update.sh` entirely:

```suggestion
  passthru.updateScript = nix-update-script { };
```

---

Ahhh I see, I missed the `VERGEN_BUILD_DATE` variable. Maybe `--subpackage` would help?

```
  -s SUBPACKAGE, --subpackage SUBPACKAGE
                        Attribute of a subpackage that nix-update should try to update hashes for
```

It's probably not worth the effort. Thanks for checking this though!

---

Yeah it's not worth it, I tried a lot of things, but it's still giving me the same error even with subpackage. That said, thank you too for the suggestion this, it might be useful for the future.